packages/peregrine/lib/talons/Header/useCartTrigger.js

```diff
@@ -33,6 +33,7 @@ export const useCartTrigger = props => {
     const fetchCartDetails = useAwaitQuery(getCartDetailsQuery);
 
     const itemCount = data ? data.cart.total_quantity : 0;
+    const hideCartTrigger = ['/checkout'].includes(history.location.pathname);
```

Comment:
I could have gone for a simple checkout page check instead of the array, but after my discussion with UX and @jimbo, I have realized this is can be a configurable option hence made an array to represent routes on which the cart trigger should be hidden.

Reply:
I'd probably leave this up to the parent component for reuseability.

Reply:
Until we decouple cart creation, this would be tricky; this component needs to render. Someone refreshing from this page would never get their cart init'd; or someone signing in from the checkout page would also skip the init of their cart. Both edges, but definitely things Dev would find in QA 😂

Reply:
You're right. I guess until we move cart init out of cart trigger the component must always render (even if hidden).